runtime/j9vm/j7vmi.c

```diff
@@ -2709,7 +2709,7 @@ JVM_DefineClassWithSourceCond(jint arg0, jint arg1, jint arg2, jint arg3, jint a
 jobject JNICALL
 JVM_EnqueueOperation(jint arg0, jint arg1, jint arg2, jint arg3, jint arg4)
 {
-	assert(!"JVM_EnqueueOperation() stubbed!");
+	assert(!"An OpenJDK Attach API may have been used to connect to an OpenJ9 VM. This is not supported.");
```

Comment:
I think this needs to mention Hotspot, otherwise it is unclear what it means because both Hotspot and OpenJ9 use OpenJDK.

Reply:
Would it be more clear to say: "A HotSpot JVM Attach API is attempting to connect to an OpenJ9 VM. This is not supported."?

Reply:
Works for me. Although you can use "HotSpot VM" instead of "HotSpot JVM"